scripts/installDependencies.sh

```diff
@@ -19,34 +21,25 @@ install_node_version() {
     if [ -s "$nvm_path" ];
     then
         echo "NVM is already installed."
+        source $nvm_path
     else
         echo "Installing NVM..."
         install_nvm
-        source nvm_path
-    fi
-
-    node_command=`command -v node`
-    current_node_version=`node -v |  awk -F\. '{print $1}'`
-    # Check if correct version of node is present
-    if [[ "$node_command" ]] && [[ "$node_version" == "$current_node_version" ]];
-    then
-        echo "Correct node version is already installed."
-    else
-        echo "Node version $node_version is not installed. Installing..."
-        nvm install $node_version
-        nvm use $node_version
+        source $nvm_path
     fi
+    
+    echo "Installing Node version $node_version..."
+    nvm install $node_version
+    nvm use $node_version
```

Comment:
Optional, running `nvm use` after `nvm install` is redundant. `nvm install` already ensures that the current installed version is used.
```suggestion
    nvm install $node_version
```

Reply:
Done.